src/Npgsql/NpgsqlConnector.cs

```diff
@@ -1244,7 +1241,7 @@ internal void CloseOngoingOperations()
 
             try
             {
-                currentCopyOperation.Dispose();
+                ((IDisposable)currentCopyOperation).Dispose();
```

Comment:
Why cannot `ICancelable` be `IDisposable`?

Reply:
It can, but that doesn't logically follow from it (not everything cancelable must necessarily be disposable).

But the actual reason is that Rider complains about implementing IDisposable twice, once directly on the type and once via ICancelable. I don't have any strong feelings here, if you really don't like it I'll change back.

Reply:
Your logic is correct, but not in the case of internal interfaces and classes. They are for infrastructure things only and for us, so why not to make code more friendly?

Reply:
OK, reverted since you feel strongly about this.